.circleci/unittest/scripts/environment.yml

```diff
@@ -7,8 +7,11 @@ dependencies:
   - pytest
   - pytest-cov
   - codecov
-  - librosa
+  - librosa #[not win]
   - pip
   - pip:
-    - kaldi-io
     - scipy
+    - kaldi-io #[win]
+    - PySoundFile #[win]
+    - librosa #[win]
+    - future #[win]
```

Comment:
Well, I don't know exactly this is needed. But without this, an error will be thrown.
```
Processing dependencies for torchaudio==0.6.0a0+b7648ac
Searching for future
Reading https://pypi.org/simple/future/
Downloading https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz#sha256=b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d
error: Download error for https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz#sha256=b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)
```

Reply:
Thanks for noting this